tests/test_pg_specific.py

```diff
@@ -594,30 +595,28 @@ def test_pg_toolbar_channels_plus_minus(raw_orig, pg_backend):
     if fig.mne.butterfly is True:
         fig._fake_keypress("b")  # toggle butterfly off
 
-    for _ in range(10):
-        if ymax - ymin <= 2:
-            break
+    while ymax - ymin > 2:  # while more than 1 channel visible
```

Comment:
I actually prefer the old pattern... `while` is dangerous and can cause nasty timeouts (could be like 80 min on CI etc.!) and the old `for` will not. Even better would be:
```
for ...
    if ...
        ...
        break
else:
    raise RuntimeError("Condition not met!")  # pragma: no cover
```

Reply:
Then we have to bump the number of iterations quite a bit, because 10 were only sufficient if you go in steps of 10. I'm not sure this is really a problem, and to be honest, I don't really understand what this does other than breaking out of the for loop before it is exhausted.

Reply:
Bumping the loop to 100 or even 1000 seems fine

Reply:
I understand the point about preferring a for loop over a while loop, but can we be more explicit then? The goal of this part of the test is to reduce the number of visible channels from 20 to 1, so this requires exactly 19 iterations. What's the point of having 100 and breaking out prematurely?